docs/src/pages/components/autocomplete/ComboBox.tsx

```diff
@@ -5,14 +5,17 @@ import Autocomplete from '@material-ui/core/Autocomplete';
 
 export default function ComboBox() {
   return (
-    <Autocomplete
-      id="combo-box-demo"
-      options={top100Films}
-      style={{ width: 300 }}
-      renderInput={(params) => (
-        <TextField {...params} label="Combo box" variant="outlined" />
-      )}
-    />
+    <div>
+      <Autocomplete
+        disablePortal
```

Comment:
If we go down this road, I think that we should document the downsides of not using a portal. We could also include a comment about it here.